src/task_scl_eff_pot_hab.py

```diff
@@ -21,7 +20,7 @@ class SCLPolygons(SCLTask):
         "probability": {
             "ee_type": SCLTask.IMAGECOLLECTION,
             "ee_path": "projects/SCL/v1/Panthera_tigris/geographies/Sumatra/hab/probability",
-            "maxage": 1, 
+            "maxage": 1,
         },
         "historic_range": {
```

Comment:
after bringing in the new `task_base` this should point to `projects/SCL/v1/Panthera_tigris/aoi` and then we rasterize as part of task